source/PlayState.hx

```diff
@@ -1,18 +1,83 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
+import flixel.group.FlxGroup;
+import flixel.text.FlxText;
+import flixel.util.FlxColor;
 
 class PlayState extends FlxState
 {
+	var screensaverGroup = new FlxTypedGroup<ScreensaverHero>();
+	var wallGroup = new FlxTypedGroup<Wall>();
+	var hero:ControllableHero;
+	var wall1:Wall;
+	var wall2:Wall;
+	var wall3:Wall;
+	var wall4:Wall;
+	var screensaver:ScreensaverHero;
+	var screensaver1:ScreensaverHero;
+	var screensaver2:ScreensaverHero;
+	var screensaver3:ScreensaverHero;
+	var screensaver4:ScreensaverHero;
+	var score:Int = 0;
+	var scoreText:FlxText;
+
 	override public function create()
 	{
 		super.create();
-		var ourHero:Hero = new Hero(300, 300, "John");
-		add(ourHero);
+		trace(FlxG.height);
+		trace(FlxG.width);
```

Comment:
these probably aren't necessary anymore, I assume they were added for your own reference during dev.